testing/fzf/APKBUILD

```diff
@@ -0,0 +1,44 @@
+# Contributor: Kevin Daudt <ops@ikke.info>
+# Maintainer: Kevin Daudt <ops@ikke.info>
+pkgname=fzf
+_pkgver="0.17.0-2"
+pkgver=${_pkgver/-/.}
+pkgrel=0
+pkgdesc="A command-line fuzzy finder"
+url="https://github.com/junegunn/fzf"
+arch="all"
+license="MIT"
+makedepends="go"
+subpackages="$pkgname-tmux"
+source="$pkgname-$_pkgver.tar.gz::https://github.com/junegunn/fzf/archive/$_pkgver.tar.gz"
+builddir="$srcdir/$pkgname-$_pkgver"
+
+build() {
+	cd "$builddir"
+	make
```

Comment:
It fetches some dependencies from Internet using `go get`, i.e. without any proper verification. That’s forbidden. Use glide with lock file.

Reply:
Is there any guide how to properly do this? Or a package that shows how it's done?